bundles/org.openhab.io.homekit/src/main/resources/OH-INF/config/config.xml

```diff
@@ -22,7 +22,11 @@
 			<label>Thermostat Current Heating/Cooling Mapping</label>
 			<description>String values used by your thermostat to set different targetHeatingCooling modes</description>
 		</parameter-group>
-
+		<parameter name="qrCode" type="text" required="false" groupName="core">
+			<label>HomeKit QR Code</label>
+			<context>qrcode</context>
```

Comment:
This context is not documented here https://www.openhab.org/docs/developer/bindings/config-xml.html#xml-structure-for-configuration-descriptions Are you sure it exists?